backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewCreateService.java

```diff
@@ -565,17 +571,83 @@ public boolean isEmptyAcceptActionUsagePayload(Element element) {
         return result;
     }
 
-    public Element createSuccessionEdge(ActionUsage sourceAction, ActionUsage targetAction) {
-        Element sourceParentElement = sourceAction.getOwner();
+    public Element createSuccessionEdge(Element successionSource, Element successionTarget) {
+        if (this.utilService.isStandardStartAction(successionSource)) {
+            return this.createSuccessionEdge(successionSource, successionTarget, successionSource.eContainer());
```

Comment:
May be you could just store the `successionOwner` in a local variable and have only one return calling`this.createSuccessionEdge`?
Could you also please add a small comment to tell that in case of a standard start action the `successionSource` is a Membership and that is why you use `successionSource.eContainer()` instead of  `successionSource.getOwner()`.

Reply:
Done